pandas/core/strings.py

```diff
@@ -68,29 +76,29 @@ def str_cat(arr, others=None, sep=None, na_rep=None):
     When ``na_rep`` is `None` (default behavior), NaN value(s)
     in the Series are ignored.
 
-    >>> Series(['a','b',np.nan,'c']).str.cat(sep=' ')
+    >>> pd.Series(['a','b',np.nan,'c']).str.cat(sep=' ')
```

Comment:
Can you split this example in two steps?

s = pd.Series(...) 
s.str.cat(sep=..)

Reply:
Yes, I can do this way as well.